src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBlockControl/SimpleText_MaxLines_Two_With_Wrap.xaml

```diff
@@ -16,26 +16,17 @@
 	d:DesignWidth="400">
 
 	<StackPanel>
-		<Slider Minimum="10" Maximum="1600" Value="350" x:Name="slider">
-			<Slider.Header>
-				<TextBlock>Container Width <Run Text="{Binding Value, ElementName=slider}" /></TextBlock>
-			</Slider.Header>
-		</Slider>
-		<Slider Minimum="5" Maximum="300" Value="70" x:Name="sliderV">
-			<Slider.Header>
-				<TextBlock>Container Height <Run Text="{Binding Value, ElementName=sliderV}" /></TextBlock>
-			</Slider.Header>
-		</Slider>
-		<Slider Minimum="5" Maximum="170" Value="20" x:Name="fontsize">
-			<Slider.Header>
-				<TextBlock>Font Size <Run Text="{Binding Value, ElementName=fontsize}" /></TextBlock>
-			</Slider.Header>
-		</Slider>
-		<Slider Minimum="5" Maximum="170" Value="20" x:Name="lineheight">
-			<Slider.Header>
-				<TextBlock>Line Height <Run Text="{Binding Value, ElementName=lineheight}" /></TextBlock>
-			</Slider.Header>
-		</Slider>
```

Comment:
Isn't this block still useful ?

Reply:
Issue https://github.com/unoplatform/uno/issues/4974 has been created to address this